src/proj.h

```diff
@@ -182,27 +262,93 @@ PJ_OBSERVATION pj_apply (PJ *P, enum pj_direction direction, PJ_OBSERVATION obs)
 /* Measure internal consistency - in forward or inverse direction */
 double pj_roundtrip(PJ *P, enum pj_direction direction, int n, PJ_OBSERVATION obs);
 
-
 int pj_show_triplet (FILE *stream, const char *banner, PJ_TRIPLET point);
 
-/* Constructor for the OBSERVATION object */
+PJ *pj_pj (PJ_CONTEXT *ctx, char *definition);
```

Comment:
I agree with 1 & 2 and changed trhe name to pj_create which I believe strikes a balance between brevity and expressivity :-)